1_fetch.R

```diff
@@ -36,6 +36,40 @@ p1_targets_list <- list(
     sf::st_as_sf(p1_AOI,coords=c("lon","lat"),crs=4326) %>%
       summarize(geometry = st_combine(geometry)) %>%
       sf::st_cast("POLYGON")
+  ),
+  
+  # Create a big grid of boxes to set up chunked data queries
+  tar_target(
+    p1_conus_grid,
+    tigris::states(class = "sf", year = 2020, progress_bar = FALSE) %>%
```

Comment:
I like the idea of this grid code and the code for getting the grid that intersects with the aoi to be inside their own script just to clean things up. I know it seems that I may be a bit all over the place with my feedback about whether to put code in a script or leave here, but I tend to sweep code into a custom function when it is more than 3 lines of code or so. I also do this when I think the code itself is not super understandable to a user and I think I can make a function name that is more helpful/descriptive.

Reply:
I appreciate your input here, @lindsayplatt! I agree that having the code within a function cleans up the phase file and I like your benchmarks for when to use a separate function. I've been trying to think about what would be most easily understood/interpreted for anyone looking at the repo, even if they don't have a lot of experience with `targets`. I also like the idea of moving the code to build these two targets to separate functions because these code steps are somewhat tangential to the main task of downloading data from WQP. This way a user isn't as likely to get bogged down thinking they need to interpret the code (unless they want to dig into the functions further).

Reply:
That's a great plan for this repo - separate the parts that are tangential to the main idea into their own functions and showcase the code behind the main targets. I like that!